backend/dal/Migrations/20210106182851_v01.06.01.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
```

Comment:
As you'll need to update `v01.06.00` from merging `v01.05.03` you may as well merge this back into `v01.06.00` too